project/src/components/film-card-promo/film-card-promo.tsx

```diff
@@ -22,7 +22,7 @@ function FilmCardPromo({Promofilm, className=''}: FilmCardPromoProps): JSX.Eleme
             <span className="film-card__year">{releaseDate}</span>
           </p>
 
-          <FilmButtons />
+          <FilmButtons currentFilm={Promofilm} />
```

Comment:
Почему с большой буквы?
```suggestion
          <FilmButtons currentFilm={promofilm} />
```